java/ql/lib/change-notes/2022-10-17-android-activity-alias.md

```diff
@@ -0,0 +1,8 @@
+---
+category: minorAnalysis
+---
+* Added the `AndroidActivityAliasXmlElement` class to represent `<activity-alias>` components in Android manifests.
```

Comment:
I would avoid being so detailed in the change note (I know it’s consistent with previous PRs, but it’s getting out of hand) and instead focus on the added functionality. In this case, I would mention something along the lines of “Data flow sources are now properly recognized in exported Android activity aliases when they target a not exported activity”. Similarly with permissions if we actually implement that as well.